suppress/suppress.go

```diff
@@ -63,6 +63,11 @@ func WrapFor(depth int, duration time.Duration, id string, f func(int, string))
 
 	tag := fmt.Sprintf("%s:%d %s", filepath.Base(file), line, id)
 
+	if duration <= 0 {
+		f(1, tag)
+		return
```

Comment:
If we wanted to allow someone to pass in 0, there are two things we could do.  One, would be to drop in `atomic.StoreInt64(&state.count, 0)` right before we return here.  This introduces a "race" where two very parallel calls might collapse into one.  It's not obvious to me if calling a "Suppress" function with a 0 duration should do this or not.  But if the intent of calling "Suppress" with 0 is that every call get called, instead of this you should just make the top of the function:
```go
func WrapFor(depth int, duration time.Duration, id string, f func(int, string)) {
 	pc, file, line, _ := runtime.Caller(depth)
 	key := locKey{pc, id}
	if duration <= 0 {
		f(1, fmt.Sprintf("%s:%d %s", filepath.Base(file), line, id))
		return
	}
	...
```